### PR TITLE
Add basic UI

### PR DESCRIPTION
**Pull request by mershark**
In this task I accomplished the following:
- Created a console app with basic UI for a school library.
- Implemented a main entry-point in app.rb for listing books, people, creating persons (students or teachers), books, rentals, and listing rentals for a given person ID.
- Ensured lint-free code, followed Gitflow, and maintained professional documentation.
- Adhered to Ruby best practices.
- Modeled the console app behavior after the provided UI example.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    age = gets.chomp
+    print 'Name: '
+    name = gets.chomp
+
+    case option
+    when '1'
+      print 'Has parent permission? [Y/N]: '
+      permission = gets.chomp.downcase
+      @people << Student.new(age, name, parent_permission: (permission == 'y'))
+    when '2'
```
Comment (omar-muhamad) [inline app.rb]: - [x] There is an error raised while creating a new student, Please fix that.

image

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+        puts "#{index}) [Teacher] Name: \"#{person.name}\", ID: #{person.id}, Age: #{person.age}"
+      end
+    end
+    person_index = gets.chomp.to_i
+
+    print 'Date (YYYY/MM/DD): '
+    date = gets.chomp
+
+    @rentals << Rental.new(date, @books[book_index], @people[person_index])
+    puts 'Rental Created Successfully'
```
Comment (omar-muhamad) [inline app.rb]: - [x] There is an error raised while creating a new rental, please fix that.

image

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @mershark,

### Nice job, you are almost there.

cat-driving-serious

## (Highlights) Good Points: :+1:
- Your work is well documented. ✔
- The correct workflow is used.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below ⬇️.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by t-yanick — APPROVED** [review]
# STATUS: MILESTONE APPROVED :1st_place_medal: 

Hi @mershark , 

### Great job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**